corehq/apps/export/export.py

```diff
@@ -433,13 +434,13 @@ def __record_datadog_export(duration, doc_bytes, n_rows, metric, tags):
     if doc_bytes:
         datadog_histogram(
             '{}_normalized_by_size'.format(metric),
-            duration / doc_bytes,
+            duration // doc_bytes,
```

Comment:
I don't think this is integer division. `duration` is a float in [at least some cases](https://github.com/dimagi/commcare-hq/blob/a0d8cc7c4109f2d349f2225a57cc39139565889f/corehq/apps/export/export.py#L413). Same below.

Reply:
It looks to me like ```duration``` is an integer because ```_time_in_milliseconds``` returns an ```int```.  Or am I missing something?

Reply:
You are correct. I didn't dig deep enough.